src/bio/bio_xstream.c

```diff
@@ -292,15 +403,19 @@ bio_nvme_init(const char *storage_path, const char *nvme_conf, int shm_id,
 
 	env = getenv("VOS_BDEV_CLASS");
 	if (env && strcasecmp(env, "MALLOC") == 0) {
-		D_WARN("Malloc device will be used!\n");
+		D_WARN("Malloc device(s) will be used!\n");
 		nvme_glb.bd_bdev_class = BDEV_CLASS_MALLOC;
 		nvme_glb.bd_bs_opts.cluster_sz = (1ULL << 20);
 		nvme_glb.bd_bs_opts.num_md_pages = 10;
 		size_mb = 2;
 		bio_chk_cnt_max = 32;
 	} else if (env && strcasecmp(env, "AIO") == 0) {
-		D_WARN("AIO device will be used!\n");
+		D_WARN("AIO device(s) will be used!\n");
 		nvme_glb.bd_bdev_class = BDEV_CLASS_AIO;
+	} else if (env && strcasecmp(env, "VMD") == 0) {
```

Comment:
I don't see when VOS_BDEV_CLASS should be set as VMD, this looks not useful to me.

Reply:
get set on the go side and set in iosrv commandline environment

Reply:
@NiuYawei its useful for the control plane, and thought it would be useful to use here just as an additional informative log message, but there will be no additional bdev class for VMD.

Reply:
ok that's fine, I was just going to use if already there, nothing critical to have if not needed in the control plane.